src/dvclive/dvc.py

```diff
@@ -107,6 +107,14 @@ def make_dvcyaml(live):
     if plots:
         dvcyaml["plots"] = plots
 
+    if live._models:
+        dvcyaml["artifacts"] = {}
+        for path, attrs in live._models.items():
+            attrs["path"] = path
+            dvcyaml["artifacts"][attrs.get("name")] = {
+                k: v for k, v in attrs.items() if k != "name"
+            }
+
```

Comment:
Why are we not keying `_models` by `name`? Then I think this becomes a one-line like `dvcyaml["artifacts"] = live._models`.

Reply:
> Why are we not keying _models by name? Then I think this becomes a one-line like dvcyaml["artifacts"] = live._models.

I had it like that and then had to change it, but I don't recall the reason 😅 . 
I will take an extra look